percona_tests/metrics_test.go

```diff
@@ -219,7 +219,7 @@ func testResolution(t *testing.T, resolutionEp, resolutionName string) {
 	missingLabelsCount := 0
 	missingLabels := ""
 	for _, oldMetric := range oldMetricsCollection.MetricsData {
-		if oldMetric.name == "" || strings.HasPrefix(oldMetric.name, "# ") {
+		if oldMetric.name == "" || strings.HasPrefix(oldMetric.name, "# ") || oldMetric.name == "go_memstats_gc_cpu_fraction" {
```

Comment:
nit: how about extracting "go_memstats_gc_cpu_fraction" to a const?

Reply:
why skipping?

Reply:
because that metric is not used within our dashboards, and it is disabled by default in new exporter